api/desecapi/serializers.py

```diff
@@ -227,7 +227,10 @@ class UserSerializer(djoserSerializers.UserSerializer):
     class Meta(djoserSerializers.UserSerializer.Meta):
         fields = tuple(User.REQUIRED_FIELDS) + (
             User.USERNAME_FIELD,
+            'dyn',
+            'limit_domains',
         )
+        read_only_fields = ('dyn', 'limit_domains',)
```

Comment:
These fields are missing documentation; also I'm considering to remove/alter the dyn flag before we make the API stable. It's seems a odd category for users and I think should be replaced with something more general. The flag is currently used to determine which domain names are acceptable for any given user to register and in order to determine which emails to send out.

Reply:
done